test.py

```diff
@@ -0,0 +1,6 @@
+from pyomeca import Angles, Rototrans
+
+angles = Angles.from_random_data(size=(1, 1, 100))
+rt = Rototrans.from_euler_angles(angles, angle_sequence="x")
```

Comment:
I think this should ideally be directly done in the Rototrans method if you call the from_random_data

Reply:
Oups, I accidentally added this file with `git add .`. Deleted in https://github.com/pyomeca/pyomeca/pull/88/commits/97051efaaedaf7e6abafa20190191606fa26b9b2